src/natcap/invest/ndr/ndr_core.pyx

```diff
@@ -129,9 +127,9 @@ cdef class _ManagedRaster:
         self.block_xbits = numpy.log2(self.block_xsize)
         self.block_ybits = numpy.log2(self.block_ysize)
         self.block_nx = (
-            self.raster_x_size + (self.block_xsize) - 1) / self.block_xsize
+            self.raster_x_size + (self.block_xsize) - 1) // self.block_xsize
```

Comment:
Was relying on python 2 style integer division. Now we need to explicitly use floor division.